apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
```

Comment:
Are we sure of the browsers that we want to support have vh and vmin support in them?
canisuse.com can help in checking if a certain browser you would like to support is compatible with vh and vmin.

Reply:
I checked on caniuse.com, and there is only partial support by IE9 and iOS7, will use a workaround instead, thanks.